torch/onnx/symbolic_opset9.py

```diff
@@ -2248,33 +2248,57 @@ def topk(g, self, k, dim, largest, sorted, out=None):
 
 
 def to(g, self, *args):
-    # ONNX doesn't have a concept of a device, so we ignore device casts
-    if len(args) == 4:
-        if args[0].node().kind() == "prim::device" or args[0].type().isSubtypeOf(ListType.ofInts()):
+
+    def is_aten_to_device_only(*args):
```

Comment:
nit: slightly clearer to get rid of the `*args` and just use `args`.
Callers don't actually need to pass in a comma-separated list of args to this func, it can always be a tuple.

Reply:
This is a nested function and I had matched the parent function signature. This is not meant to be used outside the scope of the parent function

Reply:
The call-site will also be shorter and simpler if we remove the `*` since it won't have to use the `*` at the call-site to unpack the args tuple.
i.e., it becomes `is_aten_to_device_only(args)`.